tmt/steps/execute/internal.py

```diff
@@ -302,11 +313,11 @@ def _run_tests(self, guest, extra_environment=None):
         self.debug("Pull the plan data directory.", level=2)
         guest.pull(source=self.step.plan.data_directory)
 
-    def results(self):
+    def results(self) -> List[Any]:
```

Comment:
Now we know `self._results` is `List[Result]`, therefore there's no need for `Any` in return value annotation, and we can be more specific.